samples/saga/simple/Core_6/Sample/Program.cs

```diff
@@ -16,23 +14,14 @@ static void Main()
     {
         Console.Title = "Samples.SimpleSaga";
         var endpointConfiguration = new EndpointConfiguration("Samples.SimpleSaga");
-        endpointConfiguration.UseSerialization<JsonSerializer>();
```

Comment:
this is in keeping with https://docs.particular.net/samples/#technology-choices-serializer